wp-parsely.php

```diff
@@ -91,16 +91,14 @@ function maybe_load_plugin() {
 add_action( 'muplugins_loaded', __NAMESPACE__ . '\maybe_load_plugin' );
 
 function maybe_disable_some_features() {
-	global $parsely;
-
-	if ( null != $parsely ) {
+	if ( $GLOBALS['parsely'] && is_a( $GLOBALS['parsely'], 'Parsely' ) ) {
```

Comment:
This may throw a notice, it would be safer to use an isset check
```suggestion
	if ( isset( $GLOBALS['parsely'] ) && is_a( $GLOBALS['parsely'], 'Parsely' ) ) {
```

Reply:
I originally had this and I applied some IDE suggestion. Nice catch!